services/core-web/src/components/Forms/MajorProject/MajorProjectSearchForm.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from "react";
+import { UpOutlined, DownOutlined } from "@ant-design/icons";
+import { Field, reduxForm } from "redux-form";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Button, Col, Row } from "antd";
+
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
```

Comment:
how much work would it be to take out the old inputs and the old antd compatible Form/css file?

Reply:
The old antd form has been taken out from here now.

Reply:
`renderConfig` has been replaced with the Render options from @mds/common now.